src/RulesDeploy.ts

```diff
@@ -81,22 +86,22 @@ RulesDeploy.prototype = {
             clc.bold(resourceName)
         );
       });
-  },
+  }
 
-  _compileRuleset: function(filename, files) {
+  _compileRuleset(filename: any, files: any): Promise<any> {
     utils.logBullet(
       clc.bold.cyan(this.type + ":") +
         " checking " +
         clc.bold(filename) +
         " for compilation errors..."
     );
-    var self = this;
-    return gcp.rules.testRuleset(self.options.project, files).then(function(response) {
+    const self = this;
+    return gcp.rules.testRuleset(self.options.project, files).then((response: any) => {
       if (response.body && response.body.issues && response.body.issues.length > 0) {
-        var warnings = [];
-        var errors = [];
-        response.body.issues.forEach(function(issue) {
-          var issueMessage =
+        const warnings: any = [];
```

Comment:
These are really `[]any`, right? (`warnings` and `errors`)

Reply:
done